src/Aspire.Dashboard/Otlp/WebApplicationExtensions.cs

```diff
@@ -0,0 +1,55 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Dashboard.Otlp.Grpc;
+using Microsoft.AspNetCore.Mvc;
+using OpenTelemetry.Proto.Collector.Logs.V1;
+using OpenTelemetry.Proto.Collector.Metrics.V1;
+using OpenTelemetry.Proto.Collector.Trace.V1;
+
+namespace Aspire.Dashboard.Otlp;
+
+public static class WebApplicationExtensions
+{
+    public static WebApplication ConfigureHttpOtlp(this WebApplication app, Uri httpEndpoint)
+    {
+        app.MapPost("/v1/metrics",
+            async ([FromServices] OtlpMetricsService service, HttpContext httpContext) =>
+            {
+                var body = await ReadFullyAsync(httpContext).ConfigureAwait(false);
+                var metrics = ExportMetricsServiceRequest.Parser.ParseFrom(body);
+                await service.Export(metrics, null!).ConfigureAwait(false);
+            }
+        ).AllowAnonymous()
```

Comment:
Shouldn't this use the otlp auth policy?

https://github.com/dotnet/aspire/blob/03c6a503930f0a37a089a05fa82b49e636b261dd/src/Aspire.Dashboard/Otlp/Grpc/OtlpMetricsService.cs#L14

Reply:
Good catch!
changed to 
`.RequireAuthorization(OtlpAuthorization.PolicyName)`